misc/releaser.py

```diff
@@ -537,7 +598,48 @@ def build_main(args: argparse.Namespace) -> None:
     # TODO: rethink the non-release checks
     # current_version = get_version_from_repo_describe_tag(args.verbose)
     # check_non_release(current_version)
-    build_release(args.version, yes=args.yes)
+    version: Version = args.version
+    yes: bool = args.yes
+    base_ref: str | None = args.base_ref
+
+    if version.is_dev:
+        raise ReleaseError(f"Don't release a dev version: {version}")
+    release_branch = get_release_branch(version)
+    tag = version.with_v_prefix()
+
+    print(f"Building release {COLOR_GREEN}{version}{COLOR_END} ...")
+    print(f"Will use the branch {COLOR_GREEN}{release_branch}{COLOR_END} as a release branch.")
+    print(f"Will annotate the release commit with the tag {COLOR_GREEN}{tag}{COLOR_END}.")
```

Comment:
I would prefer more direct messages (subjective opinion ^^):
```suggestion
    print(f"Release version to build: {COLOR_GREEN}{version}{COLOR_END}")
    print(f"Release branch: {COLOR_GREEN}{release_branch}{COLOR_END}")
    print(f"Release commit tag: {COLOR_GREEN}{tag}{COLOR_END}")
```